lib/ethereum/http_client.rb

```diff
@@ -2,14 +2,15 @@
 require 'json'
 module Ethereum
   class HttpClient < Client
-    attr_accessor :host, :port, :uri, :ssl
+    attr_accessor :host, :port, :uri, :ssl, :proxy
 
-    def initialize(host, log = false)
+    def initialize(host, proxy = nil, log = false)
```

Comment:
`proxy` argument should have been appended to the existing arguments. This forces everyone to update their code.

Otherwise they get an exception such as:
```NoMethodError: undefined method `gsub' for true:TrueClass```

Reply:
That is a good point. We can also use named parameter(s).